java/ql/src/semmle/code/java/frameworks/jackson/JacksonSerializability.qll

```diff
@@ -50,6 +50,15 @@ library class JacksonWriteValueMethod extends Method, TaintPreservingCallable {
   }
 }
 
+library class JacksonReadValueMethod extends Method, TaintPreservingCallable {
```

Comment:
The [language reference](https://codeql.github.com/docs/ql-language-reference/annotations/#library) says that the `library` annotation is deprecated.
It probably does not matter much here whether the class has this annotation so maybe the annotation could be omitted?

Reply:
The rest of this file uses `library`. Should I remove it from the rest of the file? I was going for consistency.

Reply:
I just changed all references in the file to `private`